src/Components/PageNotFound.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+const PageNotFound = () => {
+	return(
+		<div>
+			<h1>404 Page Not Found </h1>
+		</div>
+	)
+}
+
+export default PageNotFound
```

Comment:
This file is redundant since we covered it with the ErrorLanding page, so this can be deleted